kong/plugins/oauth2/access.lua

```diff
@@ -1,22 +1,27 @@
 local url = require "socket.url"
 local utils = require "kong.tools.utils"
-local responses = require "kong.tools.responses"
 local constants = require "kong.constants"
 local timestamp = require "kong.tools.timestamp"
-local public_utils = require "kong.tools.public"
+
 
 local string_find = string.find
-local req_get_headers = ngx.req.get_headers
-local ngx_set_header = ngx.req.set_header
-local ngx_req_get_method = ngx.req.get_method
-local ngx_req_get_uri_args = ngx.req.get_uri_args
+
+
+local split = utils.split
+local strip = utils.strip
```

Comment:
E.g. Here we keep relying on `strip`. Also, not sure why we are localizing keys of the `utils` module? No big deal though.

Reply:
`strip`== `require "pl.stringx".strip` (so maybe change to PL dependency?)
`split`== `require "pl.stringx".split` (so maybe change to PL dependency?)